sdk/communication/azure-communication-identity/tests/conftest.py

```diff
@@ -0,0 +1,92 @@
+# --------------------------------------------------------------------------
+#
+# Copyright (c) Microsoft Corporation. All rights reserved.
+#
+# The MIT License (MIT)
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the ""Software""), to
+# deal in the Software without restriction, including without limitation the
+# rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+# sell copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+# IN THE SOFTWARE.
+#
+# --------------------------------------------------------------------------
+
+# cSpell:ignore ests
+import pytest
+import os
+from devtools_testutils import test_proxy, add_general_regex_sanitizer, add_header_regex_sanitizer, \
+    set_default_session_settings, add_body_key_sanitizer, add_oauth_response_sanitizer
+from azure.communication.identity._shared.utils import parse_connection_str
+
+@pytest.fixture(scope="session", autouse=True)
+def add_sanitizers(test_proxy):
+    set_default_session_settings()
+    add_oauth_response_sanitizer()
+
+    connection_str = os.getenv('COMMUNICATION_LIVETEST_DYNAMIC_CONNECTION_STRING')
+    if connection_str is not None:
+        endpoint, _ = parse_connection_str(connection_str)
+        resource_name = endpoint.split(".")[0]
+        add_general_regex_sanitizer(regex=resource_name, value="sanitized")
```

Comment:
There are some cases where complex strings (like connection strings) have caused issues when being used with a regex sanitizer. It looks like you're sanitizing a simplified value instead of the full connection string, but I would recommend using `add_general_string_sanitizer` in place of the regex sanitizer for environment variables since we know that these variables will have plain string values

Reply:
Thanks! I'll update.